### ISSUE TITLE
Strange packaging result: Four items in four containers

### ISSUE THREAD

**Issue by shuairan** [skjolber/3d-bin-container-packing#574]
Hey, for the following example the packager packs every box in a single container:

```  
@Test
public void testFourForTwo() throws Exception {
	List<Container> containers = new ArrayList<>();
	containers.add(Container.newBuilder().withDescription("1").withEmptyWeight(0).withSize(60, 40, 25).withMaxLoadWeight(100).build());

	LargestAreaFitFirstPackager packager = LargestAreaFitFirstPackager.newBuilder().withContainers(containers).build();

	StackableItem b1 = new StackableItem(Box.newBuilder().withId("b1").withDescription("b1").withSize(35, 25, 14).withWeight(0).withRotate3D().build(), 1);
	StackableItem b2 = new StackableItem(Box.newBuilder().withId("b2").withDescription("b2").withSize(35, 25, 14).withWeight(0).withRotate3D().build(), 1);
	StackableItem b3 = new StackableItem(Box.newBuilder().withId("b3").withDescription("b3").withSize(35, 23, 13).withWeight(0).withRotate3D().build(), 1);
	StackableItem b4 = new StackableItem(Box.newBuilder().withId("b4").withDescription("b4").withSize(35, 25, 14).withWeight(0).withRotate3D().build(), 1);

	List<Container> packaging = packager.packList(Arrays.asList(b1, b2, b3, b4), 100);
	write(packaging);
}
```

When Box **b3**  has `withSize(35, 23, 14)`  (dz +1) it packs the four boxes into two containers. That seems to be kind of strange. Is that a bug or misbehaviour in LAFF?

**Comment by shuairan** [comment]
Visualisation
![574_3d-skjober_four_containers](https://user-images.githubusercontent.com/213402/195553485-ace5f37d-4134-49a8-9288-9982e576d6be.png)

**Comment by skjolber** [comment]
Fixed in 2.1.4
